app/src/main/java/nerd/tuxmobil/fahrplan/congress/details/EventDetailFragment.java

```diff
@@ -159,7 +164,22 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
             if (lecture != null && lecture.dateUTC > 0) {
                 DateFormat df = SimpleDateFormat
                         .getDateTimeInstance(SimpleDateFormat.SHORT, SimpleDateFormat.SHORT);
-                t.setText(df.format(new Date(lecture.dateUTC)) + " - " + room);
+
+                Drawable drawable_ACESS_TIME = getContext().getResources().getDrawable(R.drawable.ic_access_time);
```

Comment:
Please stick to Java naming conventions: `drawable_ACESS_TIME` -> `drawableAcessTime`. Snackcase is used in Python. Java used camelcase.
Further, I recommend moving the type to the end: `accessTimeDrawable`.

Reply:
I was trying to use the java [constant convention](https://www.oracle.com/technetwork/articles/javaee/codeconventions-135099.html) here but I failed as wanted to easily identify the constant name and the `Drawable` abstract class name so i used that convention
![Screen Shot 2019-11-01 at 19 14 02](https://user-images.githubusercontent.com/845598/68046276-cedae480-fcdb-11e9-860b-b381c6d8377c.png)

Reply:
Constants come with the `static` keyword in Java.